### ISSUE TITLE
Stays on black screen on PS2

### ISSUE THREAD

**Issue by Insidious849** [filipalac/240pTestSuite-PS1#15]
I'm trying to use this on my PS2 with a USB and Free McBoot so that I can calibrate my PS2 picture for my games.

I followed all of the instructions on how to create the folders, extract this, extract that, convert this .cue file to a .VCD; everything. When I try to play it via ULaunchElf, the PlayStation startup screen appears, but then, I get a black screen afterwards.

What is the problem? I tried the regular version and the version for emulators. Do I have to use a modchip to make it work? I really don't want to have to do that and risk effing up my console. I use an original fat/phat PS2 console.

**Comment by ramapcsx2** [comment]
No, this homebrew is known to sometimes black screen at boot, even on original hardware.
If several retries don't work, it's probably not compatible with your PS2 :/

**Comment by BonQerss** [comment]
Having this exact same issue on PS2.

**Comment by TEC10101** [comment]
Try this fixed version, it worked to fix this issue for my Xstation https://github.com/filipalac/240pTestSuite-PS1/files/7532930/240P-PS1-fixed-Alcahest.zip
